_posts/pl/2021-12-17-frontend-developer.md

```diff
@@ -0,0 +1,157 @@
+
+Udział w projekcie Mailbox oferuje zrównoważoną dawkę złożoności biznesowej i wyzwań technologicznych.
+
+Dodatkowym atutem jest możliwość pochwalenia się znajomym i rodzinie co się ostatnio robiło, wystarczy, że są klientami jednego z naszych klientów, a że klienci duzi to i szansa na bycie ich klientem spora ;-)
+
+~45k linii TypeScript z ~8k linii SASS stojących na froncie przed klastrem Kafki i Solara brzmi kusząco? Mailbox jest dla Ciebie!
+
+### Eximee
+
+Praca z domeną Cię nie kręci? Zawsze od tematów biznesowych bardziej kręciły Cię wyzwania czysto technologiczne?
+
+Może warto spojrzeć na nasz system obsługi wniosków i procesów? Eximee na każdym kroku projektowane jest jako framework, z którego nasi klienci korzystają budując własne rozwiązania biznesowe.
+
+W Eximee nigdy nie ma jednego słusznego rozwiązania problemu, wszystko jest warunkowe, "zależy" i "oczywiście, że można zrobić inaczej". Podstawą pracy jest, z jednej stronie naginanie tego co założyli twórcy, a z drugiej projektowanie systemu podatnego na naginanie. Skala systemu leży w jego generyczności i rozszerzalności. Samo Eximee jest uruchomione w kilkunastu wariantach wdrożeniowych u różnych klientów, często w wielu kanałach równolegle!
+
+Wszystko to sprawia, że Eximee to głównie wyzwanie technologiczne i poligon nowych rozwiązań testowanych u klientów w publicznie dostępnych systemach!
+
+Ale to nie wszystko, w projekcie budujemy też narzędzia graficznego projektowania procesów i wniosków, obsługi backoffice i inne. Poznaj potrzeby systemu spełniającego oczekiwania od klienta, przez obsługę backoffice, po pracę programistów oraz projektantów tworzących biznesowe rozwiązania oparte o dostarczaną przez Ciebie platformę.
+
+Twoi znajomi pewnie już korzystali z systemu który tworzysz, zapewne nie raz ;-)
+
+Być może Eximee z ~250k linii TypeScript i ~130k linii SASS frameworku to coś dla Ciebie.
+
+### Consdata OSS
+
+Gdy nabierzesz ochoty na eksperymenty, prototypu i ogólnie luźniejsze podejście, zawsze możesz zaplanować swój czas na rozwój lub budżet chapterowy na wsparcie jednego z naszych projektów open source skupionych na GitHub: [https://github.com/orgs/Consdata/repositories](https://github.com/orgs/Consdata/repositories).
+
+Tylko z nazwy warto wspomnieć o:
+
+- [kouncil](https://github.com/Consdata/kouncil) - ui do Kafki zbudowany na bazie doświadczeń naszych i naszych klientów,
+- [sonarqube-companion](https://github.com/Consdata/sonarqube-companion) - agregat naruszeń SonarQube rozumiejący strukturę zespołów i śledzący trendy analizy statycznej,
+- inne narzędzia jak logger, aplikacje intranetowe do newsletterów, rezerwacji parkingu, czy śledzenia feedbacku i zbierania ankiet.
+
+W Open Source nie ograniczamy się żadnymi wytycznymi, to nasze poligony. Firebase? Angular czy React? Node czy Java? Spring czy Quarkus? Nie ma znaczenia! Zrealizuj funkcjonalność i powiedz innym jak poszło!
```

Comment:
dałabym już bez ostatniego wykrzyknika:
_W Open Source nie ograniczamy się żadnymi wytycznymi, to nasze poligony poznańskiej doliny krzemowej ;). Firebase? Angular czy React? Node czy Java? Spring czy Quarkus? Nie ma znaczenia! Zrealizuj funkcjonalność i powiedz innym jak poszło._